modules/crio_get.adoc

```diff
@@ -0,0 +1,117 @@
+= Getting CRI-O
+CRI-O is not supported as a stand-alone container engine.
+CRI-O is always meant to be used as the container engine for a Kubernetes
+installation, such as OpenShift. To run containers without Kubernetes
+or OpenShift, use link:https://github.com/containers/libpod/tree/master/cmd/podman[podman].
+
+To set up a CRI-O
+container engine to use with an {product-title} cluster, you can:
+
+* Install CRI-O along with a new {product-title} cluster or
+* Add a node to an existing cluster and identify CRI-O as the container engine for
+that node. Both CRI-O and Docker nodes can exist on the same cluster.
+
+The following sections describe those ways of getting CRI-O.
+
+== Installing CRI-O with a new {product-title} cluster
+Choosing CRI-O as the container engine for your {product-title} nodes can be done on
+a per-node basis at {product-title} install time. Here are a few things you should
+know about enabling the CRI-O container engine when you install {product-title}:
+
+* Previously, using CRI-O on your nodes required that the docker container engine be
+available as well, to perform build and push operations to the registry. As of
+{product-title} 3.10 and later, the docker service is no longer required. So you can now have
+CRI-O-only nodes in your {product-title} cluster.
+
+* Enabling CRI-O using a CRI-O enginer container is no longer supported. An
+rpm-based installation of CRI-O is required.
+
+The following procedure assumes you are installing {product-title} using Ansible
+inventory files, such as those described in
+link:https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html-single/installing_clusters/#install-config-configuring-inventory-file[Configuring Your Inventory File].
+
+[NOTE]
+====
+Do not set `/var/lib/docker` as a separate mount point for an {product-title} node
+using CRI-O as its container engine. When deploying a CRI-O node, the installer tries
+to make `/var/lib/docker` a symbolic link to `/var/lib/containers`. That action
+will fail because it won't be able to remove the existing `/var/lib/docker` to
+create the symbolic link.
+====
+
+. With the {product-title} Ansible playbooks installed, edit the appropriate
+inventory file to enable CRI-O. For example, when installing a single-node
+cluster on local host, edit the `hosts.localhost` file.
+
```

Comment:
@vikram-redhat What is the `hosts.localhost` file? I am not a Linux expert, but I don't see the file and don't see anything on the Internet. Maybe @chrisnegus can help?